src/rlookup.c

```diff
@@ -599,10 +599,25 @@ static RSValue *replyElemToValue(RedisModuleCallReply *rep, RLookupCoerceType ot
   }
 }
 
+static bool isValueAvailable(const RLookupKey *kk, const RLookupRow *dst, RLookupLoadOptions *options)
+{
+  if (!options->forceLoad && kk && !(kk->flags & RLOOKUP_F_FORCE_LOAD)) {
```

Comment:
`RLOOKUP_F_FORCE_LOAD` is never turned on on a key (see `RLOOKUP_TRANSIENT_FLAGS` definition)